src/httpclient.cc

```diff
@@ -54,6 +59,8 @@ HttpClient::HttpClient() : user_agent(std::string("Aktualizr/") + AKTUALIZR_VERS
   headers = curl_slist_append(headers, "Accept: */*");
   curl_easy_setopt(curl, CURLOPT_HTTPHEADER, headers);
   curl_easy_setopt(curl, CURLOPT_USERAGENT, user_agent.c_str());
+
+  pkcs11_key = pkcs11_cert = false;
```

Comment:
Would it be nicer to put this at the top of the ctor?

~~~
HttpClient::HttpClient() : 
    user_agent(std::string("Aktualizr/") + AKTUALIZR_VERSION) 
    pkcs11_key(false),
    pkcs11_cert(false) {
~~~

Reply:
Yes, sure makes sense.